src/FontStashSharp/StbTrueTypeSharpFontLoader/StbTrueTypeSharpFontSource.cs

```diff
@@ -103,6 +103,18 @@ public void RasterizeGlyphBitmap(int glyphId, int fontSize, byte[] buffer, int s
 			}
 		}
 
+		public void RasterizeGlyphBitmap(int glyphId, int fontSize, byte[] buffer, int startIndex, int outWidth, int outHeight, int outStride, uint kernelWidth)
+		{
+			UpdateSize(fontSize);
+
+			fixed (byte* output = &buffer[startIndex])
+			{
+				stbtt_MakeGlyphBitmap(_font, output, outWidth, outHeight, outStride, Scale, Scale, glyphId);
+				stbtt__h_prefilter(output, outWidth, outHeight, outStride, kernelWidth);
+				stbtt__v_prefilter(output, outWidth, outHeight, outStride, kernelWidth);

```

Comment:
I would suggest to add a `kernelHeight` argument because horizontal prefilter (`kernelWidth`) is usually used with 3 for best quality and vertical prefilter (`kernelHeight`) is usually 2.

Reply:
Very good point, I'll adjust this tomorrow evening (working the whole day)